pkg/util/paramtable/component_param.go

```diff
@@ -2939,6 +2942,31 @@ func (p *dataNodeConfig) init(base *BaseTable) {
 	}
 	p.MemoryForceSyncSegmentNum.Init(base.mgr)
 
+	p.MemoryForceSyncSegmentMinNum = ParamItem{
+		Key:          "datanode.memory.forceSyncSegmentMinNum",
+		Version:      "2.4.0",
+		DefaultValue: "1",
+	}
+	p.MemoryForceSyncSegmentMinNum.Init(base.mgr)
+
+	p.MemoryForceSyncMinSize = ParamItem{
+		Key:          "datanode.memory.forceSyncMinSize",
+		Version:      "2.3.6",
```

Comment:
What's the difference between 2.3.6 and 2.4.0 for MemoryForceSyncSegmentMinNum

Reply:
planed to add this param item in 2.3.6, removing it